Eat_and_Greet/src/screens/ChatsPage/index.tsx

```diff
@@ -0,0 +1,27 @@
+import { View } from 'react-native';
+import ChatCard from '../../comps/ChatCard/ChatCard'; 
+
+import { Toolbar } from '../../comps/Toolbar/toolbar';
+
+import styles from '../../style';
+
+function HomeScreen() {
+
+    return (
+      <View style={styles.ScreenContainer}>
+      <View style={styles.OuterBox}>
+  
+        <ChatCard senderName="Basketball Chat" message="Steph said: Game@2:30 tmrw" timestamp="4:45 PM" />
```

Comment:
I'm thinking we should create an optional prop to signify who said a message in a groupchat instead of it as part of the message. This is unnecessary for now though, will be needed when adding functionality

Reply:
ill make another ticket